index.html

```diff
@@ -32,12 +67,12 @@ <h1>Parallel Plots</h1>
   </div>
   <div id="wrap">
     <div class="third">
-      <p id="food-list">
+      <p id="item-list">
```

Comment:
this was the last remains of "food" referenced besides in comments. 
still need to strip out references in comments.